src/Core/Services/Implementations/UserService.cs

```diff
@@ -32,6 +32,8 @@
 using File = System.IO.File;
 using JsonSerializer = System.Text.Json.JsonSerializer;
 
+#nullable enable
```

Comment:
This is probably enabling nullable for too much of the file - check out all of the Github annotations below in the diff view. Only enable nullable on code that expects it. (i.e. you've written or reviewed it to make sure it handles nullability properly)